core/dataset_classifier.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+from core.db_client import DBClient
+
+
+class DatasetClassifier(object):
+	
+	def __init__(self):
+		"""
+		Initalizes instance of Dataset Classifier. Only 
+		"""
+		self.db_client = DBClient()
```

Comment:
I think it's a better idea to use dependency injection here as there's a parameter that can be set on the DBCLient class. It will be useful if we ever need to write tests that test the DatasetClassifier with a test DBClient.

Reply:
I agree. Updated.